packages/react-charting/src/components/HorizontalBarChartWithAxis/HorizontalBarChartWithAxis.types.ts

```diff
@@ -0,0 +1,130 @@
+import { IStyle } from '@fluentui/react/lib/Styling';
+import { IRenderFunction, IStyleFunctionOrObject } from '@fluentui/react/lib/Utilities';
+import {
+  ICartesianChartProps,
+  ICartesianChartStyleProps,
+  ICartesianChartStyles,
+  IHorizontalBarChartWithAxisDataPoint,
+} from '../../index';
+
+export interface IHorizontalBarChartWithAxisProps extends ICartesianChartProps {
+  /**
+   * Data to render in the chart.
+   */
+  data?: IHorizontalBarChartWithAxisDataPoint[];
+
+  /**
+   * Define a custom callout renderer for a data point.
+   */
+  onRenderCalloutPerDataPoint?: IRenderFunction<IHorizontalBarChartWithAxisDataPoint>;
+
+  /**
+   * Width of each bar in the chart.
+   */
+  barWidth?: number;
```

Comment:
this will be called barHeight for this chart

Reply:
Done